.github/workflows/linux-ubsan.yml

```
@@ -22,8 +22,9 @@ jobs:
           submodules: true
       - uses: ./.github/actions/build-cmake-preset
         with:
-          preset-name: clang-relwithdebinfo
-          cmake-args: "-D SILKIT_ENABLE_UBSAN=ON -D SILKIT_BUILD_DASHBOARD=OFF \
+          preset-name: relwithdebinfo
+          cmake-args: "-D SILKIT_BUILD_DASHBOARD=OFF \
                        -DCMAKE_C_COMPILER=clang-18 \
-                       -DCMAKE_CXX_COMPILER=clang++-18"
+                       -DCMAKE_CXX_COMPILER=clang++-18 \
+                       -DCMAKE_CXX_FLAGS='-fsanitize=undefined -fno-omit-frame-pointer -g3'"
```

The `-g3` flags will be overwritten by the CMake provided flags, as the value of `CMAKE_CXX_FLAGS` is _prepended_ to the build-type dependent flags.

Same goes for the other workflows.

We can however simply overwrite those by hand:

```suggestion
                       -DCMAKE_CXX_FLAGS='-fsanitize=undefined -fno-omit-frame-pointer'" \
                       -DCMAKE_CXX_FLAGS_RELWITHDEBINFO='-Og -g3'"
```

---

The default for RelWithDebInfo is `-O2 -g` btw. And later occurences on the `gcc` or `clang` command line override the earlier ones.

---

The none `_INIT` version, so forget the previous comment